BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/Platform/PlatformImplementation.cs

```diff
@@ -177,17 +178,25 @@ public IDebugOutput GetDebugOutput()
         {
             if (this.debugOutput == null)
             {
-                this.debugOutput = new TelemetryDebugWriter(); 
+                this.debugOutput = new TelemetryDebugWriter();
             }
-            
+
             return this.debugOutput;
         }
 
         /// <inheritdoc />
         public bool TryGetEnvironmentVariable(string name, out string value)
         {
-            value = Environment.GetEnvironmentVariable(name);
-            return !string.IsNullOrEmpty(value);
+            try
+            {
+                value = Environment.GetEnvironmentVariable(name);
+                return !string.IsNullOrEmpty(value);
+            }
+            catch (Exception e)
+            {
+                CoreEventSource.Log.FailedToLoadEnvironmentVariables(e.ToString());
+                throw;
```

Comment:
a Try/Get method should not throw exceptions

Reply:
ok. just removed!